website/hw/hw4.rst

```diff
@@ -180,103 +218,389 @@ As you are stepping through your program, you might want to be able to
 see the values of certain variables. You can do this by running the
 ``print`` command. ``print val``, for example, will print the value
 ``val``. With pointers, you can use ``print *p`` to print out the value
-pointed to by p. If you want to print in hex, use ``print/x val``, and
+pointed to by p. If you want to print in hex, use ``print/x val``, and
 to print in binary use ``print/b val``
```

Comment:
Running `print/b p` is actually not working (though `print/x p` is). Looks like the command may be `print/t p`